config/ses.toml

```diff
@@ -6,15 +6,18 @@
 target = "devel7"
 
   [ses.ses7]
+    rook_ceph_chart = "registry.suse.de/suse/sle-15-sp2/update/products/ses7/charts/rook-ceph:latest"
     [ses.ses7.yaml_substitutions]
       # "suse.com" = "suse.de/devel/storage/7.0/containers"
 
     [ses.ses7.repositories]
       # Each key/value under this sub-heading is a repository
-      storage = "http://download.suse.de/ibs/SUSE/Products/Storage/7/x86_64/product/"
+      storage         = "http://download.suse.de/ibs/SUSE/Products/Storage/7/x86_64/product/"
```

Comment:
not related to the PR, how these url can be overridden?

Reply:
If you're using env-vars, then it should be ROOKCHECK_SES__SES7__REPOSITORIES__STORAGE.

(It's using dynaconf's notation https://www.dynaconf.com/)

Reply:
@brunoleon can you fix the comment from Kyr?

Reply:
@toabctl if you're speaking about the extra space it is fixed